packages/libs/web-common/src/user-dataset-upload-config.tsx

```diff
@@ -104,9 +107,12 @@ export const uploadTypeConfig: DatasetUploadTypeConfig<ImplementedUploadTypes> =
             </a>
             .
             <br />
-            The bigwig files you select here must be mapped to the reference
+            The bigWig files you upload here must be mapped to the reference
             genome that you select above.
             <br />
+            If you need to upload more than one file please make a compressed
+            file with all your bigWig files.
```

Comment:
i think you should specify which format of compression is supported here: i guess tgz and zip